### PR TITLE
Allow disabling packing python native datatypes in blobs

### PR DESCRIPTION
**Pull request by ixcat**

### CODE REVIEW

File: datajoint/blob.py [hunk 1]
```diff
@@ -74,8 +72,8 @@ def __init__(self, squeeze=False):
         self.protocol = None
 
     def set_dj0(self):
-        if not DJ0_ENABLE:
-            raise RuntimeError('dj0 encoding disabled by user')
+        if not config.get('enable_python_native_blobs'):
+            raise RuntimeError('v0.12+ python native blobs disabled. see also: TODO URL')
```
Comment (dimitri-yatsenko) [inline datajoint/blob.py]: `DataJointError`

File: datajoint/blob.py [hunk 2]
```diff
@@ -71,6 +72,9 @@ def __init__(self, squeeze=False):
         self.protocol = None
 
     def set_dj0(self):
+        if not config.get('enable_python_native_blobs'):
+            raise DataJointError('v0.12+ python native blobs disabled. see also: https://github.com/datajoint/datajoint-python/blob/master/README.md')
```
Comment (dimitri-yatsenko) [inline datajoint/blob.py]:
```suggestion
            raise DataJointError('v0.12+ python native blobs disabled, see the 0.12.0 release notes at: https://github.com/datajoint/datajoint-python/blob/master/README.md')
```

Comment (ixcat) [reply datajoint/blob.py]: Well.. these aren't actually the release notes, but a special section in the README.
alternate suggestion?

Comment (dimitri-yatsenko) [reply datajoint/blob.py]: Where is it? Are you planning to add in this PR?

Comment (ixcat) [reply datajoint/blob.py]: It's in the docs-parts PR - can merge the pr's if that's better

Comment (dimitri-yatsenko) [reply datajoint/blob.py]: ok, merging and will sort in the next PR

**Pull request merged**